torch/csrc/autograd/saved_variable.cpp

```diff
@@ -140,8 +140,10 @@ Variable SavedVariable::unpack(std::shared_ptr<Node> saved_for) const {
 }
 
 const char* ERR_BACKWARD_TWICE =
-    "Trying to backward through the graph a second time, but the saved intermediate "
-    "results have already been freed. Specify retain_graph=True when calling "
-    ".backward() or autograd.grad() the first time.";
+    "Trying to access saved variables that have already been freed (possibly while "
```

Comment:
I think we want to have it the other way around here: You're trying to backward through the graph a second time (or directly access saved variable after they've been freed).
Because most users will reach this because of the backward twice, not because they're accessing saved variables by hand.